fetch_resources.sh

```diff
@@ -314,6 +314,7 @@ function configure_tempest
     no_of_computes=$($OPENSTACK_CMD compute service list | awk "/ nova-compute / " | wc -l)
 
     iniset $TEMPEST_CONFIG compute image_ref $image_uuid
+    iniset $TEMPEST_CONFIG compute image_ref_alt $image_uuid_alt
```

Comment:
Is this variable getting used anywhere ?

Reply:
not now, will remove